native/cocos/audio/android/UrlAudioPlayer.cpp

```diff
@@ -133,20 +133,23 @@ void UrlAudioPlayer::stop() {
     ALOGV("UrlAudioPlayer::stop (%p, %d)", this, getId());
     SLresult r = (*_playItf)->SetPlayState(_playItf, SL_PLAYSTATE_STOPPED);
     SL_RETURN_IF_FAILED(r, "UrlAudioPlayer::stop failed");
+    {
+          std::lock_guard<std::mutex> guard(destoryMutex); // 加入互斥锁
```

Comment:
Where is `destoryMutex` defined? And did you mean stop function is invoked more than once, then cause the crash?

Reply:
Oh, I forgot to upload it; in UrlAudioPlayer.h, it is a member variable.

Reply:
you can try

Reply:
How to reproduce it?

Reply:
I analyzed it from the logs, and it is intermittent, occurring in the background

Reply:
How did you know it is cause by not use mutex?